elements/lisk-p2p/src/peer/base.ts

```diff
@@ -237,7 +237,7 @@ export class Peer extends EventEmitter {
 			const rate = this._getMessageRate(message);
 			const messageWithRateInfo = {
 				...message,
-				peerId: this._id,
+				peerId: this.id,
```

Comment:
probably this doesn't need to change. Prefer to use private property over public one internally